CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 ### Chore & Maintenance
 
+* `[jest-circus]` Add dependency on jest-each ([#6309](https://github.com/facebook/jest/pull/#6309))
```

Comment:
This is now part of master, not `23.0.1` 😅

Reply:
Fixed :)